cmake/Futility_Configurations.cmake

```diff
@@ -251,7 +251,7 @@ ELSEIF(CMAKE_Fortran_COMPILER_ID STREQUAL "GNU")
     IF(${PROJECT_NAME}_FPE_TRAP)
         SET(Fortran_FLAGS
             ${Fortran_FLAGS}
-            ${CSYM}ffpe-trap=invalid,zero,overflow,underflow,denormal
+            ${CSYM}ffpe-trap=invalid,zero#,overflow,underflow,denormal
```

Comment:
Its unlikely we can handle all of these checks in one swoop.

Reply:
No the PR has no real reason to keep this, this was for my convenience and to make sure anyone that follows behind me is certain to be on the same configuration.